migrations/0xx.construction-materials.up.sql

```diff
@@ -0,0 +1,38 @@
+-- Launch of Constuction category
+-- Fields: Core construction material, Secondary construction materials, Roof covering materials
+
+-- Construction materials: Wood, Stone, Brick, Steel, Reinforced Concrete, Other Metal
+--   Other Natural Material, Other Man-Made Material
+CREATE TYPE construction_materials
+    AS ENUM ('Wood',
+    'Stone',
+    'Brick',
+    'Steel',
+    'Reinforced Concrete',
+    'Other Metal',
+    'Other Natural Material',
+    'Other Man-Made Material');
+
+-- Roof covering materials: Slate, Clay Tile, Wood, Asphalt, Iron or Steel, Other Metal
+--   Other Natural Material, Other Man-Made Material
+CREATE TYPE roof_covering
+    AS ENUM ('Slate',
+    'Clay Tile',
+    'Wood',
+    'Asphalt',
+    'Iron or Steel',
+    'Other Metal',
+    'Other Natural Material',
+    'Other Man-Made Material');
+
```

Comment:
We've been discussing with @tomalrussell  that for classifications which aren't really a gold standard, but rather something we come up with - it might be better to use reference tables to define the list of acceptable values, rather than defining an `ENUM`. This is because enums are harder to change at a later date, and with those fluid types of classifications there might be a need for adding/removing options later on. This is as opposed to something like listing grade, where we most probably will never go beyond `I`, `II*` and `II` so enum is appropriate there.
However, to avoid additional work, I'd stick with `ENUM` here. Worst case, if we need to, we can swap the underlying mechanism later.

Reply:
Agreed - happy to start with the `ENUM`